### ISSUE TITLE
Problème Apple Silicon M2

### ISSUE THREAD

**Issue by lo6816** [michael-dm/eva#7]
Bonjour,

J'ai rencontré plusieurs soucis lors de l'installation du projet eva. Pour la plupart d'entre eux, les erreurs affichées dans le command prompt étaient assez claires et il n'y a pas eu de problème pour la résolution des erreurs. Cependant une autre reste non résolue.

Pour exécuter le programme, j'ouvre un terminal à la racine du projet et je lance la commande 'pnpm dev'. Le programme se lance bien, je peux effectuer une requête orale et la formulation est bien comprise. Cependant je vois une nouvelle erreur dans le terminal : 

<img width="988" alt="Capture d’écran 2023-08-06 à 12 31 41" src="https://github.com/michael-dm/eva/assets/77900650/854ac35a-ccec-42ad-9f40-9f1e3a116583">

Si je comprends bien, le taux de requêtes envoyées à l'API est trop grand par rapport à ce qu'elle peut supporter. Est-ce à cause de la puce M2 ou juste une mauvaise utilisation de ma part ?

Si quelqu'un aurait une réponse ça serait top !

Merci d'avance

**Comment by alphaleadership** [comment]
tu envoie juste trop de requéte

**Comment by lo6816** [comment]
Mais comment limiter le nombre de requête envoyée ? D'autant plus que je ne pose qu'une seule question

**Comment by alphaleadership** [comment]
question complexe ou pas

**Comment by Sheld0n-Cooper** [comment]
Intéressant, as tu réussi ?

**Comment by Haapax** [comment]
Pour que le projet fonctionne correctement, il faut avoir un compte OpenAI et avoir des crédits disponibles sur l'API (pas la même chose que de payer ChatGPT+). 

![image](https://github.com/michael-dm/eva/assets/49695921/46c36c0c-02b4-4d35-aaf5-a4a847f10efe)

**Comment by lo6816** [comment]
Effectivement, le problème venait simplement du fait que je n'avais pas de crédit sur mon compte OpenAI. En me renseignant un petit peu, j'ai compris que mon taux de requête était trop élevé car, sans crédit, le taux limite était de 0.. Merci pour vos réponses !
